Update R script

jeroenwo opened the pull request:
An extra R script is required: doAzureParallel now requires the rAzureBatch package. If not, the following error occurs:
ERROR: dependency 'rAzureBatch' is not available for package 'doAzureParallel'

azurecla commented:
Hi __@elfisher__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (elfish). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

jeroenwo committed:
Update R script

An extra R script is required: doAzureParallel now requires the rAzureBatch package. If not, the following error occurs:
ERROR: dependency 'rAzureBatch' is not available for package 'doAzureParallel'
